src/axom/slic/docs/sphinx/sections/appendix.rst

```diff
@@ -0,0 +1,24 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/appendix:
+
+Appendix
+---------
+
+ .. _SlicApplicationCodeExample:
+
+Slic Application Code Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Below is the complete :ref:`SlicApplicationCodeExample` presented in
+the :ref:`sections/getting_started` section. The code can be found in the Axom
+source code under ``src/axom/slic/examples/basic/logging.cpp``.
+
+ .. literalinclude:: ../../../examples/basic/logging.cpp
```

Comment:
It might be helpful to add file-based logging to the example.  This might help show who is responsible for opening and closing the fstream, and when that happens relative to `slic::initialize()` and `slic::finalize()`.